gitlab/cli.py

```diff
@@ -395,37 +400,39 @@ def main() -> None:
     gitlab_resource = args.gitlab_resource
     resource_action = args.resource_action
     skip_login = args.skip_login
+    unmask_credentials = args.unmask_credentials
 
     args_dict = vars(args)
     # Remove CLI behavior-related args
     for item in (
-        "gitlab",
+        "api_version",
         "config_file",
-        "verbose",
         "debug",
+        "fields",
+        "gitlab",
         "gitlab_resource",
-        "resource_action",
-        "version",
+        "job_token",
+        "oauth_token",
         "output",
-        "fields",
+        "pagination",
+        "private_token",
+        "resource_action",
         "server_url",
+        "skip_login",
         "ssl_verify",
         "timeout",
-        "api_version",
-        "pagination",
+        "unmask_credentials",
```

Comment:
(will need more sorting if that was your idea in this PR)

```suggestion
        "mask_credentials",
```